src/Microsoft.DotNet.Build.CloudTestTasks/ListAzureBlobs.cs

```diff
@@ -102,6 +102,10 @@ public override bool Execute()
                 while (!string.IsNullOrEmpty(nextMarker))
                 {
                     urlListBlobs = string.Format($"https://{AccountName}.blob.core.windows.net/{ContainerName}?restype=container&comp=list&marker={nextMarker}");
```

Comment:
Seems pretty questionable that this url's being generated from scratch in two places--with different (and redundant, here) formatting code, too. Will be great to remove this entirely with https://github.com/dotnet/buildtools/issues/1816.

Reply:
@dagood Yeah.  The APIs are much better to use.